test Fixture with speci@l chars/MyRouteBuilderWithAdditionalDependencies.java

```diff
@@ -1,9 +1,10 @@
 // camel-k: dependency=mvn:org.apache.commons:commons-math3:3.6.1
 
 import org.apache.camel.builder.RouteBuilder;
+import org.apache.commons.math3.util.ArithmeticUtils;
 
 public class MyRouteBuilderWithAdditionalDependencies extends RouteBuilder{
 	public void configure() {
-        ArithmeticUt
+        ArithmeticUtils dummy;
```

Comment:
need to use a simpler use case due to regression between kamel local and Camel Jbang dependency copy: https://issues.apache.org/jira/browse/CAMEL-19917